Remove references to FreeSans font, which is not packaged for Guix.

--- a/game/fonts/fonts.ini	1970-01-01 01:00:01.000000000 +0100
+++ b/game/fonts/fonts.ini	2022-09-16 14:31:51.483096847 +0200
@@ -116,26 +116,6 @@
 BoldHighResPreCache=0
 BoldHighResOutline=0.02
 
-[Font_FreeSans]
-Name=Free Sans
-RegularFile=FreeSans/FreeSans.ttf
-RegularFallbackFile1=wqy-microhei/wqy-microhei.ttc
-;RegularGlyphSpacing=1.4
-;RegularStretch=1.2
-BoldFile=FreeSans/FreeSansBold.ttf
-BoldFallbackFile1=wqy-microhei/wqy-microhei.ttc
-;BoldEmbolden=0.06
-OutlineFile=FreeSans/FreeSansBold.ttf
-OutlineOutline=0.06
-;OutlineColorR=0.3
-;OutlineColorG=0.3
-;OutlineColorB=0.3
-;OutlineColorA=0.3
-BoldHighResFile=FreeSans/FreeSansBold.ttf
-BoldHighResMaxResolution=256
-BoldHighResPreCache=0
-BoldHighResOutline=0.02
-
 [Font_DejaVuSans]
 Name=DejaVu Sans
 RegularFile=DejaVu/DejaVuSans.ttf
